src/main/java/org/primefaces/component/fileupload/FileUpload.java

```diff
@@ -60,6 +61,9 @@
 
     public static final String CONTAINER_CLASS_SIMPLE = "ui-fileupload-simple ui-widget";
     public static final String FILENAME_CLASS = "ui-fileupload-filename";
+    public static final String DROPZONE_CLASS = "ui-fileupload-dropzone";
+
+    private Boolean hasDropZoneFacet;
```

Comment:
avoid local variables in components for MyFaces ViewPooling

Reply:
I've seen that in several components. Should they all be removed?

Just  some random examples:

https://github.com/primefaces/primefaces/blob/4f85a65ff583bf5823df877566210e811d54e6c6/src/main/java/org/primefaces/component/datepicker/DatePickerBase.java#L44

https://github.com/primefaces-extensions/primefaces-extensions/blob/48d5750dd1f5efd38a9e67d4d873aab60c4fcee8/core/src/main/java/org/primefaces/extensions/component/counter/Counter.java#L66

Reply:
yep, that should be avoided if not to performance risky. Otherwise resetted in saveState.

Reply:
@jepsar We can do those others in a separate PR.  For now just only fix your issue for this PR.

Reply:
Yeah, I was planning to go over the ones in PFE in a separate PR

Reply:
https://github.com/primefaces-extensions/primefaces-extensions/pull/392